### PR TITLE
Simple refactoring

### PR DESCRIPTION
**Pull request by bonustrack**
Doing a simple refactoring (see commit list) to get things working with my workspace and make it easier to get started with the repo.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,10 @@
+.DS_Store
+package-lock.json
```
Comment (Sekhmet) [inline .gitignore]: Why we need this?

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (Sekhmet) [inline .prettierrc]: Hurts to see first PR to just ignore someone's codestyle and force your own 🙄 
It works, why change it.

File: .gitignore (hunk 1 above)
Comment (bonustrack) [reply .gitignore]: I'm sure if we dont do this someone will commit a `package-lock.json` file by mistake at some point.

Comment (Sekhmet) [reply .gitignore]: Same can happen with any other file.
Missing file with 10K lines of code is pretty big mistake, I think we would notice it.

File: .prettierrc (hunk 2 above)
Comment (bonustrack) [reply .prettierrc]: Single quote and comma tangle is what we use in our current repos, it's not my codestyle it's Busy codestyle.

Comment (bonustrack) [reply .prettierrc]: I will revert it

**Review by Sekhmet — APPROVED** [review]

**Pull request merged**